.gitignore

```diff
@@ -1,3 +1,5 @@
 /vendor/
 /composer.lock
 .phpunit.result.cache
+.php-cs-fixer.cache
```

Comment:
to keep with the convention above, I would recommend either removing the leading `/` or adding it here (this is a tiny nitpick feel free to ignore it)